src/Elmish.WPF/BindingVmHelpers.fs

```diff
@@ -699,10 +699,9 @@ type Set(value: obj) =
     match binding with
     | BaseVmBinding b -> this.Base(model, b)
     | Cached b ->
-        let successful = this.Recursive(model, b.Binding)
-        if successful then
-          b.Cache.Value <- None  // TODO #185: write test
-        successful
+        // Set only dispatches a message and while UpdateModel changes the model,
+        // so don't clear the cache here
```

Comment:
Grammatically unclear. "While" would work, but "and while" doesn't when used like that. Maybe also add a period at the end.

Reply:
Or do you mean `while UpdateModel`? I'm assuming you're not using a `while`-loop, though.

Reply:
Thanks for the review.

I reworded. Is that clear now?

Reply:
Much better, thanks!